price/pkg/settings-template.toml

```diff
@@ -0,0 +1,56 @@
+# log settings for the application (RUST_LOG format). Default below
+# 
+# log = "price=debug"
+
+# RPC Endpoint for price oracles. Required.
+rpc_endpoint = "https://api.devnet.solana.com"
+
+# Price sink roll time (mins). Default = 3 mins. Optional.
+sink_roll_mins = 3
```

Comment:
move to constant.. it's not done like this in other oracles

Reply:
Fixed in [fe83739](https://github.com/helium/oracles/pull/400/commits/fe837396d363ec7094be174464b50e8b89359488)